i18n_generate.mll

```diff
@@ -81,17 +81,25 @@ and parse_string_2 buffer = parse
 
 {
 
+let print_type fmt langs =
+  Format.fprintf fmt
+    "[%%shared type t = %a]\n"
```

Comment:
The generated code gives `[%shared type t = ...]`, not `[%%shared type t = ...]`.